nautobot/core/testing/views.py

```diff
@@ -744,16 +745,20 @@ def test_table_with_indentation_is_removed_on_filter_or_sort(self):
             self.user.is_superuser = True
             self.user.save()
 
-            if not issubclass(self.model, TreeModel):
+            if not issubclass(self.model, (TreeModel)) or self.model is not Prefix:
```

Comment:
Should we use duck typing here? 

```py
if not hasattr(self.model, "ancestors"):
```

Reply:
That would match

```py
<class 'nautobot.dcim.models.device_components.InventoryItem'>
<class 'nautobot.dcim.models.devices.ControllerManagedDeviceGroup'>
<class 'nautobot.dcim.models.locations.LocationType'>
<class 'nautobot.dcim.models.locations.Location'>
<class 'nautobot.dcim.models.racks.RackGroup'>
<class 'nautobot.ipam.models.Prefix'>
<class 'nautobot.ipam.models.IPAddress'>
<class 'nautobot.tenancy.models.TenantGroup'>
```

Reply:
Looks like that's causing the latest CI failure?